hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -318,10 +319,13 @@ int SaraNcpClient::off() {
         return SYSTEM_ERROR_INVALID_STATE;
     }
     muxer_.stop();
-    // Disable voltage translator
-    modemSetUartState(false);
-    // Power down
-    modemPowerOff();
+    // Try using AT command to turn off the modem first.
+    if (modemSoftPowerOff() != 0) {
+        // Disable voltage translator
+        modemSetUartState(false);
```

Comment:
Even with soft power off you need to disable it, otherwise you'll be leaking current through it.

Reply:
It does if sending the AT command successfully.